2019-04-06 13:56:45 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Diagnosis.diagnosis_template_name' in 'field list' in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php on line 39
Request URL: /admin/diagnosis
Referer URL: http://localhost/pms/admin/diagnosis/add
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(39): PDOStatement->execute(NULL)
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Connection.php(311): Cake\Database\Statement\MysqlStatement->execute()
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Query.php(213): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#3 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\ORM\Query.php(1011): Cake\Database\Query->execute()
#4 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Datasource\QueryTrait.php(276): Cake\ORM\Query->_execute()
#5 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\ORM\Query.php(960): Cake\ORM\Query->_all()
#6 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Component\PaginatorComponent.php(191): Cake\ORM\Query->all()
#7 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php(717): Cake\Controller\Component\PaginatorComponent->paginate(Object(Cake\ORM\Query), Array)
#8 D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\DiagnosisController.php(67): Cake\Controller\Controller->paginate(Object(Cake\ORM\Query))
#9 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php(440): App\Controller\Admin\DiagnosisController->index()
#10 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(119): Cake\Controller\Controller->invokeAction()
#11 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\Admin\DiagnosisController))
#12 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#13 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#14 {main}


2019-04-06 13:57:08 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Diagnosis.diagnosis_template_name' in 'field list' in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php on line 39
Request URL: /admin/diagnosis
Referer URL: http://localhost/pms/admin/diagnosis/add
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(39): PDOStatement->execute(NULL)
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Connection.php(311): Cake\Database\Statement\MysqlStatement->execute()
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Query.php(213): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#3 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\ORM\Query.php(1011): Cake\Database\Query->execute()
#4 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Datasource\QueryTrait.php(276): Cake\ORM\Query->_execute()
#5 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\ORM\Query.php(960): Cake\ORM\Query->_all()
#6 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Component\PaginatorComponent.php(191): Cake\ORM\Query->all()
#7 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php(717): Cake\Controller\Component\PaginatorComponent->paginate(Object(Cake\ORM\Query), Array)
#8 D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\DiagnosisController.php(67): Cake\Controller\Controller->paginate(Object(Cake\ORM\Query))
#9 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php(440): App\Controller\Admin\DiagnosisController->index()
#10 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(119): Cake\Controller\Controller->invokeAction()
#11 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\Admin\DiagnosisController))
#12 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#13 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#14 {main}


2019-04-06 13:58:22 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Diagnosis.diagnosis_template_name' in 'where clause' in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php on line 39
Request URL: /admin/diagnosis/add
Referer URL: http://localhost/pms/admin/diagnosis/add
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(39): PDOStatement->execute(NULL)
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Connection.php(311): Cake\Database\Statement\MysqlStatement->execute()
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Query.php(213): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#3 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\ORM\Query.php(1011): Cake\Database\Query->execute()
#4 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Datasource\QueryTrait.php(276): Cake\ORM\Query->_execute()
#5 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\ORM\Query.php(960): Cake\ORM\Query->_all()
#6 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Datasource\QueryTrait.php(404): Cake\ORM\Query->all()
#7 D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\DiagnosisController.php(110): Cake\ORM\Query->first()
#8 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php(440): App\Controller\Admin\DiagnosisController->add()
#9 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(119): Cake\Controller\Controller->invokeAction()
#10 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\Admin\DiagnosisController))
#11 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#12 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#13 {main}


2019-04-06 15:45:24 Error: Fatal Error (4): syntax error, unexpected '$session' (T_VARIABLE) in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UsersController.php, line 47]
Request URL: /admin/users/logout
Referer URL: http://localhost/pms/admin/users
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 223
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 105
[main] - [internal], line ??



2019-04-06 15:45:24 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '$session' (T_VARIABLE) in D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UsersController.php on line 47
Request URL: /admin/users/logout
Referer URL: http://localhost/pms/admin/users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(105): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'D:\\Xampp_5.6\\ht...', 47)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2019-04-06 21:04:27 Error: [RuntimeException] Unable to locate an object compatible with paginate. in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php on line 713
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Client IP: 127.0.0.1
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php(46): Cake\Controller\Controller->paginate(false)
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php(440): App\Controller\Admin\UpcomingUsersController->index()
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(119): Cake\Controller\Controller->invokeAction()
#3 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\Admin\UpcomingUsersController))
#4 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#5 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#6 {main}


2019-04-06 21:06:40 Error: Fatal Error (1): Call to undefined method App\Controller\Admin\UpcomingUsersController::resetDashboardSearch() in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php, line 14]
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 223
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 105
[main] - [internal], line ??



2019-04-06 21:06:40 Error: [Cake\Error\FatalErrorException] Call to undefined method App\Controller\Admin\UpcomingUsersController::resetDashboardSearch() in D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php on line 14
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(105): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'D:\\Xampp_5.6\\ht...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2019-04-06 21:07:06 Error: Fatal Error (1): Call to undefined method App\Controller\Admin\UpcomingUsersController::checkById() in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php, line 78]
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 223
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 105
[main] - [internal], line ??



2019-04-06 21:07:07 Error: [Cake\Error\FatalErrorException] Call to undefined method App\Controller\Admin\UpcomingUsersController::checkById() in D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php on line 78
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(105): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'D:\\Xampp_5.6\\ht...', 78)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2019-04-06 21:07:34 Error: Fatal Error (1): Call to a member function find() on boolean in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php, line 27]
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 223
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 105
[main] - [internal], line ??



2019-04-06 21:07:34 Error: [Cake\Error\FatalErrorException] Call to a member function find() on boolean in D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php on line 27
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(105): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'D:\\Xampp_5.6\\ht...', 27)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2019-04-06 21:17:17 Error: Fatal Error (1): Call to undefined method App\Controller\Admin\UpcomingUsersController::checkById() in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php, line 99]
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 223
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 105
[main] - [internal], line ??



2019-04-06 21:17:17 Error: [Cake\Error\FatalErrorException] Call to undefined method App\Controller\Admin\UpcomingUsersController::checkById() in D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\UpcomingUsersController.php on line 99
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(105): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'D:\\Xampp_5.6\\ht...', 99)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2019-04-06 21:21:50 Error: [Cake\Controller\Exception\MissingActionException] Action UpcomingUsersController::edit() could not be found, or is not accessible. in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php on line 430
Exception Attributes: array (
  'controller' => 'UpcomingUsersController',
  'action' => 'edit',
  'prefix' => 'admin',
  'plugin' => false,
)
Request URL: /admin/upcoming-users/edit/186
Referer URL: http://localhost/pms/admin/upcoming-users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(119): Cake\Controller\Controller->invokeAction()
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\Admin\UpcomingUsersController))
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#3 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#4 {main}


2019-04-06 21:31:26 Error: [Cake\Controller\Exception\MissingActionException] Action UpcomingUsersController::edit() could not be found, or is not accessible. in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Controller\Controller.php on line 430
Exception Attributes: array (
  'controller' => 'UpcomingUsersController',
  'action' => 'edit',
  'prefix' => 'admin',
  'plugin' => false,
)
Request URL: /admin/upcoming-users/edit/186
Referer URL: http://localhost/pms/admin/upcoming-users
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(119): Cake\Controller\Controller->invokeAction()
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\Admin\UpcomingUsersController))
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#3 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#4 {main}


2019-04-06 22:11:40 Error: could not save user
2019-04-06 22:14:13 Error: could not save user
2019-04-06 22:15:16 Error: could not save user
2019-04-06 22:53:16 Error: [Cake\Routing\Exception\MissingControllerException] Controller class UpcomingUsers could not be found. in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ControllerFactory.php on line 89
Exception Attributes: array (
  'class' => 'UpcomingUsers',
  'plugin' => false,
  'prefix' => 'admin',
  '_ext' => false,
)
Request URL: /admin/upcoming-users
Referer URL: http://localhost/pms/admin/users/add
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(70): Cake\Http\ControllerFactory->missingController(Object(Cake\Http\ServerRequest))
#1 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Filter\ControllerFactoryFilter.php(63): Cake\Http\ControllerFactory->create(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#2 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Filter\ControllerFactoryFilter.php(49): Cake\Routing\Filter\ControllerFactoryFilter->_getController(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#3 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\DispatcherFilter.php(144): Cake\Routing\Filter\ControllerFactoryFilter->beforeDispatch(Object(Cake\Event\Event))
#4 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Event\EventManager.php(414): Cake\Routing\DispatcherFilter->handle(Object(Cake\Event\Event), Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#5 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Event\EventManager.php(391): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#6 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#7 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(78): Cake\Http\ActionDispatcher->dispatchEvent('Dispatcher.befo...', Array)
#8 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(60): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#9 D:\Xampp_5.6\htdocs\pms\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#10 {main}


2019-04-06 23:09:23 Warning: Headers already sent in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\Debugger.php:822
2019-04-06 23:59:47 Warning: Warning (2): date() expects at least 1 parameter, 0 given in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\AppointmentsController.php, line 80]
Request URL: /admin/appointments
Referer URL: http://localhost/pms/admin/appointments
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 153
date - [internal], line ??
App\Controller\Admin\AppointmentsController::__search() - APP/Controller\Admin\AppointmentsController.php, line 80
App\Controller\Admin\AppointmentsController::index() - APP/Controller\Admin\AppointmentsController.php, line 39
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 440
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 119
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 93
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 60
[main] - ROOT\webroot\index.php, line 37



2019-04-06 23:59:58 Warning: Warning (2): date() expects at least 1 parameter, 0 given in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\AppointmentsController.php, line 80]
Request URL: /admin/appointments
Referer URL: http://localhost/pms/admin/appointments
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 153
date - [internal], line ??
App\Controller\Admin\AppointmentsController::__search() - APP/Controller\Admin\AppointmentsController.php, line 80
App\Controller\Admin\AppointmentsController::index() - APP/Controller\Admin\AppointmentsController.php, line 39
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 440
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 119
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 93
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 60
[main] - ROOT\webroot\index.php, line 37



2019-04-07 00:02:36 Warning: Warning (2): date() expects at least 1 parameter, 0 given in [D:\Xampp_5.6\htdocs\pms\src\Controller\Admin\AppointmentsController.php, line 80]
Request URL: /admin/appointments
Referer URL: http://localhost/pms/admin/appointments
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 153
date - [internal], line ??
App\Controller\Admin\AppointmentsController::__search() - APP/Controller\Admin\AppointmentsController.php, line 80
App\Controller\Admin\AppointmentsController::index() - APP/Controller\Admin\AppointmentsController.php, line 39
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 440
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 119
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 93
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 60
[main] - ROOT\webroot\index.php, line 37



2019-04-07 00:02:40 Error: Fatal Error (1): Call to a member function format() on boolean in [D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Type\DateTimeType.php, line 117]
Request URL: /admin/appointments
Referer URL: http://localhost/pms/admin/appointments
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 223
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 105
[main] - [internal], line ??



2019-04-07 00:02:40 Error: [Cake\Error\FatalErrorException] Call to a member function format() on boolean in D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Database\Type\DateTimeType.php on line 117
Request URL: /admin/appointments
Referer URL: http://localhost/pms/admin/appointments
Stack Trace:
#0 D:\Xampp_5.6\htdocs\pms\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(105): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'D:\\Xampp_5.6\\ht...', 117)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


